pkg/miekg/answers.go

```diff
@@ -796,6 +796,10 @@ func ParseAnswer(ans dns.RR) interface{} {
 		return makeSVCBAnswer(&cAns.SVCB)
 	case *dns.SVCB:
 		return makeSVCBAnswer(cAns)
+	case *dns.OPT:
```

Comment:
Just curious how adding this here. Is this better off than the default case where you get _some_ data back? This seems to reduce the amount of data made accessible (unless it's triggering some error condition).

Reply:
I thought the default presentation of the OPT record was not very useful but reading [draft-peltan-edns-presentation-format-00.html](https://www.ietf.org/id/draft-peltan-edns-presentation-format-00.html), the default is not far off the example given in section 3.

Instead of reverting that change, I thought I may as well add some code to parse a basic EDNS0 answer. e.g.

```json
{
"flags": "do",
"type": "EDNS0",
"udp_size": 1232,
"version": 0
}
```

Reply:
Love it. Thanks for doing that.